components/core/tests/test_log_files/log.txt

```diff
@@ -1,3 +1,4 @@
+123123 relative timestamp
```

Comment:
Besides this, let's add some tests in `test-TimestampPattern.cpp`

Reply:
commit 7b7a022ad396a29b675609a1f39633acd47218dc

Reply:
commit 5f3b0a3b1ae0cf1a82924f4dbc0536c4448e67c1